product/alerts/rss/microsoft_vms.yml

```diff
@@ -1,44 +1,44 @@
-# Feed metadata

```

Comment:
The reason for this big diff is the newlines - believe these were Microsoft-style newlines....could that be a problem?